lib/rspec/support/ruby_features.rb

```diff
@@ -55,6 +55,18 @@ def caller_locations_supported?
         respond_to?(:caller_locations, true)
       end
 
+      begin
```

Comment:
What do you think about doing a `respond_to?` check here? I think that'd be a lot cleaner.